components/bill/HistoryTable.tsx

```diff
@@ -25,18 +26,67 @@ const BillHistoryActionRows = ({ billHistory }: HistoryProps) => {
 export const HistoryTable = ({ billHistory }: HistoryProps) => {
   return (
     <div className="text-center">
-      <Table responsive striped bordered hover>
+      <StyledTable>
         <thead>
           <tr>
             <th></th>
+            <th>Status History</th>
             <th>Branch</th>
-            <th>Action</th>
           </tr>
         </thead>
         <tbody>
           <BillHistoryActionRows billHistory={billHistory} />
         </tbody>
-      </Table>
+      </StyledTable>
     </div>
   )
 }
+
+const StyledTable = styled.table`
+  table-layout: auto;
+  width: 100%;
+
+  th {
+    font: {
+      family: Nunito;
+      size: 16px;
+      weight: 400;
+    }
+    line-height: 22px;
+    letter-spacing: -1.125px;
+    text-align: start;
+    color: white;
+    background-color: var(--bs-secondary);
+
+    padding: 0.5rem;
+    outline: 0.25rem solid white;
+  }
+
+  th:last-child {
+    width: 24%;
+    text-align: center;
+  }
+
+  tbody {
+    td {
+      font: {
+        family: Nunito;
+        size: 16px;
+        weight: 400;
+      }
```

Comment:
Not sure if these are being handled correctly by styled-components. You might have to use font-family / font-size / font-weight. I think you also need to wrap Nunito in double quotes